framework/doc/content/source/markers/ReporterPointMarker.md

```diff
@@ -0,0 +1,19 @@
+# ReporterPointMarker
+
+!syntax description /Adaptivity/Markers/ReporterPointMarker
+
+## Description
+
+The `ReporterPointMarker` is a stand-alone marker that marks all
+elements containing points defined by coordinates given in a `Reporter`.
+
+
+## Example Input Syntax
+
```

Comment:
would be nice to have a 2 line description of what is being done in the input file